hubblestack/extmods/fdg/curl.py

```diff
@@ -96,9 +90,12 @@ def request(url,
 
     chained
         Ignored
+
+    chained_status
+        Ignored
     """
-    if chained:
-        log.warn('Chained value detected in curl.request module. Chained '
+    if chained or chained_status:
```

Comment:
I was thinking if `chained_status` is true it would be the equivalent of having `chained` passed in, as it would mean another module called it, it was successfully executed and is trying to pass in a value. On a second thought now, `chained` would already narrow down that possibility so I'll remove it in the next commit